locales/it/index.json

```diff
@@ -4648,6 +4648,13 @@
               "continueAlt": "Aggiungi documento",
               "close": "Continua più tardi"
             }
+          },
+          "successL3": {
```

Comment:
What if we use `successL2` for the current flow and `success` for the new one instead? We could reduce the amount of code to refactor when we will dismiss the old flow.
What do you think?